src/main/resources/assets/computercraft/lua/rom/apis/paintutils.lua

```diff
@@ -16,37 +23,60 @@ function loadImage( sPath )
 
     local tImage = {}
     if fs.exists( sPath ) then
-        local file = io.open(sPath, "r" )
-        local sLine = file:read()
-        while sLine do
+        for sLine in io.lines(sPath) do
             local tLine = {}
-            for x=1,sLine:len() do
-                tLine[x] = tColourLookup[ string.byte(sLine,x,x) ] or 0
+            for x=1,#sLine do
+                tLine[x] = tColourLookup[ string.sub(sLine,x,x) ] or 0
```

Comment:
sub is used here so that char doesn't need to be used when rebuilding strings for paintutils.save and paintutils.drawImage.